content/millennium/r4/foundation/security/provenance.md

```diff
@@ -43,10 +49,15 @@ Search for Provenances that meet supplied query parameters:
 
 ### Parameters
 
- Name         | Required? | Type          | Description
---------------|-----------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
- `_id`        | This or `target` | [`token`]     | The logical resource id associated with the resource. Example: `doc-7845`
- `target`     | This or `_id` | [`reference`] | Target Reference(s) (usually version specific). The AllergyIntolerance, CarePlan, CareTeam, Condition, DiagnosticReport, DocumentReference, Encounter, Goal, Immunization, MedicationRequest, Observation, Patient and Procedure resources are supported. Example: `DocumentReference/66547`
+ Name      | Required?     | Type          | Description
+-----------|---------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------------
+ `_id`     | Conditionally | [`token`]     | The logical resource ID associated with the resource. It may be a list separated by commas. This parameter is required if the `target` parameter is not used. Example: `doc-1234`
+ `target`  | Conditionally | [`reference`] | The target references, which can be version-specific. The AllergyIntolerance, CarePlan, CareTeam, Condition, DiagnosticReport, DocumentReference, Encounter, Goal, Immunization, MedicationRequest, Observation, Patient, and Procedure resources are supported. Example: `DocumentReference/66547`
```

Comment:
in other places, we generally make this singular, then specify that multiple may be provided. should probably be consistent here as well.